packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.types.ts

```diff
@@ -6,6 +6,7 @@ export { Align, Justify };
 
 type ModifiedTooltipProps = Omit<
   TooltipProps,
+  | 'children'
```

Comment:
I don't think `children` is even a valid prop on `TooltipProps`, is it?

Reply:
we're overriding it as required below. it was optional before